### ISSUE TITLE
Accessibility - iOS - Screens Missing Language

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#92]
In the iOS screens, there is not even a default language set for the pages which should be English (lang="en"). By not having a language set or the wrong one will cause issues for screen reader users. Say I'm a Spanish speaking user and I come to your page that doesn't have language set. VoiceOver will read your English words with a Spanish accent really fast.

When a person changes languages to either Spanish or French, the pages should be set to that language for all pages in the application until the person switches to something else. Which most likely won't happen ever again for others or often, but because I'm testing all the languages mine should keep changing.

So no Spanish or French accents for the proper pages. VoiceOver is reading "2 / 8" as "two slash eight" on the Spanish page instead of "dos barras ocho" or "deux slash huit" for the French page. Please note my translations came from Google translate. So please get any other messages in the application appropriately translated.

**Comment by sanganalakshay** [comment]
fixed

**Comment by jfc3-dol** [comment]
VoiceOver is now reading the content in all three languages for all the different pages I tested. Was not able to look at all of them but did a random sampling and assuming they will stay with the language that was set.

I notice the words "Search Results" at the top of the search results page after the home link in both Spanish and French. Along with the words "X results found for "test" where "X" is the number of results.

Will create new issues for each of these items.

Closing this issue since it has been corrected.
